content/docs/command-reference/gc.md

```diff
@@ -1,60 +1,75 @@
 # gc
 
-Remove unused objects from <abbr>cache</abbr> or remote storage.
+Remove unused files and directories from <abbr>cache</abbr> or
+[remote storage](/doc/command-reference/remote).
 
 ## Synopsis
 
 ```usage
-usage: dvc gc [-h] [-q | -v] [-a] [-T] [-c] [-r <name>]
+usage: dvc gc [-h] [-q | -v]
+              [-w] [-a] [-T] [--all-commits] [-c] [-r <name>]
               [-f] [-j <number>] [-p [<path> [<path> ...]]]
 ```
 
 ## Description
 
-This command deletes (garbage collects) data files or directories that may exist
-in the cache (or [remote storage](/doc/command-reference/remote) if `-c` is
-used) but no longer referenced in [DVC-files](/doc/user-guide/dvc-file-format)
-currently in the <abbr>workspace</abbr>. By default, this command only cleans up
-the local cache, which is typically located on the same machine as the project
-in question. This usually helps to free up disk space.
+This command deletes (garbage collects) data files or directories that exist in
+DVC cache but are no longer needed. With `--cloud` it also removes data in
```

Comment:
> but are no longer needed.

It'd be better to express this in any other way. I don't have a better suggestion, though than the following:
```suggestion
DVC cache but are no longer in use. With `--cloud` it also removes data in
```

Reply:
Good question, I think needed was there before. I don't have a very strong opinion ... may be even `needed` better because it is more precise here - I just keep stuff that I need. need is more general then use. Still, need to review other place with in use/need.

Reply:
Changed to always use `need` vs `in-use`. I think `need` and `needed` are actually more correct here.